index.html

```diff
@@ -263,7 +264,13 @@ <h3>
             <li>Presentation Request Data Model</li>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy Guidance for Verifiable Credentials</li>
-            <li>An API for Verifiable Credential Exchange</li>
+            <li>A Developer Guide consisting of one or more notes related to:
+              <ul>
+                <li>General implementation guidance and best practices for working with VCs</li>
+                <li>One or more HTTP protocol definitions for Verifiable Credential Exchange (such as the <a href="https://w3c-ccg.github.io/vc-api/">VC-API</a>)</li>
+                <li>Guidance on Verifiable Credential Exchange over OpenID Connect</li>
```

Comment:
```suggestion
                <li>Verifiable Credential Exchange over OpenID Connect</li>
```
Suggesting some simpler language - again, the leading sentence above already implies one-or-more notes on things... so the group can decide to touch on OIDC or not. As I'm not sure what "potential guidance" means, but your explanation @mprorock is very clear... maybe this wording is more concise?

Reply:
I definitely prefer this - but want broader input from the group

Reply:
It's a good idea to call out specific APIs that the group will want to comment on. I will note that the work happening in ISO 23220 is also contemplating protocols that transmit VCs/VPs, and that should be in scope as well (since we know that it's happening).

We should also add "any other protocols that transmit VCs/VPs". It is imperative that the VCWG provides guidance to protocol developers on appropriate ways of integrating the data model into protocols. We don't want to find ourselves in the position of not being able to comment on other groups re-defining/using the data model in ways that are not appropriate.

Concrete ask: Please put providing guidance to ISO 23220-* in scope as well as any other protocol that utilizes the VCWG data models.

Reply:
@mprorock when you adjust this can you add ISO 23220-*?

Reply:
First, I am confused why "one or more HTTP protocol" is separate from "over OpenID Connect". OpenID Connect is an HTTP Protocol. It should be one line.

Reply:
Agree with @Sakurann -- we should list each protocol we're interested in commenting on -- one per line. We should finally have a "... and other relevant protocols that exchange Verifiable Credentials." (or something to that effect).

Reply:
The idea of my proposed text was to have the item be "one or more HTTP protocols" and then a list of those protocols, including VCAPI, OIDC, GNAP, and "others"